docs/guides/monitoring.md

```diff
@@ -3,6 +3,68 @@
+  protocol = "udp"
+  max_tcp_connections = 250
+  tcp_keep_alive = false
+  service_address = ":8125"
+
+  delete_gauges = true
+  delete_counters = true
+  delete_sets = true
+  delete_timings = true
+
+  percentiles = [50.0, 90.0, 99.0, 99.9, 99.95, 100.0]
+
+  metric_separator = "_"
+
+  parse_data_dog_tags = false
+  datadog_extensions = false
+  datadog_distributions = false
+
+  allowed_pending_messages = 10000
+  percentile_limit = 1000
+
+[[outputs.influxdb]]
+  urls = ["http://127.0.0.1:8086"]
+  database = "raccoon"
+  retention_policy = "autogen"
+  write_consistency = "any"
+  timeout = "5s"
+```
+
+**Configure Raccoon To Send To The Metric Collector**
+After you have the collector with the port configured, you need to set [METRIC\_STATSD\_ADDRESS](https://odpf.gitbook.io/raccoon/reference/configurations#metric_statsd_address) to match the metric collector address. Suppose you deploy the telegraf using the default configuration above as sidecar or in localhost, you need to set the value to `:8125`.
+
+**Visualize And Create Alerting From The Collected Metrics**
+Now that you have Raccoon and Telegraf as metric collector set, next is to use the metrics reported. You may notice that the Telegraf config above contains `outputs.influxdb`. That config will send the metric received to influxdb. Make sure you have influx service accessible from the configured URL.
+You can visualize the metrics using Grafana. To do that, you need to [add influx datasource](https://www.influxdata.com/blog/how-grafana-dashboard-influxdb-flux-influxql/) to make the data available on Grafana. After that you can use the data to You can visualize the metrics using Grafana. To do that, you need to [add influx datasource](https://www.influxdata.com/blog/how-grafana-dashboard-influxdb-flux-influxql/) to make the data available on Grafana. After that, you can use the data to [make dashboard](https://grafana.com/docs/grafana/latest/datasources/influxdb/#influxql-query-editor)
+.
+
+## Metrics Usages
+
+Following are key monitoring statistics that you can infer from Raccoon metrics. Refer to this section to get a clue of how to build alerting, dashboard, or do analysis on the metrics.
+### Data Loss
+To infer data loss, you can count [`kafka_messages_delivered_total`](https://odpf.gitbook.io/raccoon/reference/metrics#kafka_messages_delivered_total) with tag `success=false`. You can also infer loss rate by calculating 
+> count(`kafka_messages_delivered_total` `success=false`) / count(`kafka_messages_delivered_total`)
+### Latency
+Raccoon provides fine-grained metrics that denote latency. That gives clues as to where to look in case something goes wrong. In summary, these are key metrics for latency:
+- [`event_processing_duration_milliseconds`](https://odpf.gitbook.io/raccoon/reference/metrics#event_processing_duration_milliseconds) This metrics denotes overall latency. You need to look at other latency metrics to find the root cause when this metric is high.
```

Comment:
need to clarify what each of the latency is (which time to which time). For ex., A worker latency computes the time taken between fetching the batch from the event channel and the each event acknowledged by kafka.

Reply:
We have [an image](https://odpf.gitbook.io/raccoon/reference/metrics#event-delivery) to help the user understand when are these metrics are emitted. Will give a reference to this section be enough?

Reply:
My bad, didn't see the hyperlink, sounds good 👍